lib/js/src/compose/types/chart/base.ts

```diff
@@ -270,6 +275,41 @@ export class BaseChart {
         axisPosition: 'left',
         labelPosition: 'end',
       },
+      tooltip: {},
+      legend: {
+        position: {
+          top: '0',
+          right: '0',
+          bottom: '0',
+          left: '0',
+        },
+      },
+      pieAndDoughnutOffset: {
+        top: '0',
+        right: '0',
+        bottom: '0',
+        left: '0',
+        isDefault: true,
+        defaultValues: {
+          top: '45',
```

Comment:
dont keep defaultValues here, just hardcode them where you need them. since they are both the same